magisk/post-fs-data.sh

```diff
@@ -1,4 +1,4 @@
 #!/system/bin/sh
 
-# Remove Play Services from DenyList, otherwise the Zygisk module won't load
-magisk --denylist rm com.google.android.gms
+# Remove Play Services from the Magisk Denylist when set to enforcing.
+magisk --denylist status 2> /dev/null && magisk --denylist rm com.google.android.gms
```

Comment:
I know `&&` works as a shorthand in this case, but please use this construct for clarity:

```bash
if cmd; then
    ...
fi
```